storm-core/src/jvm/org/apache/storm/utils/Time.java

```diff
@@ -102,6 +102,14 @@ public static long currentTimeMillis() {
     public static int currentTimeSecs() {
         return (int) (currentTimeMillis() / 1000);
     }
+
+    public static int delta(int timeInSeconds) {
+        return Time.currentTimeSecs() - timeInSeconds;
+    }
+
+    public static long deltaMs(long timeInMilliseconds) {
```

Comment:
I am conflicted about this.  The function feels like it should be a part of time, but Time provides APIs that can simulate time, but this function does not.  At a minimum can we add some javadocs explaining this?